posthog/api/test/test_insight.py

```diff
@@ -2901,22 +2901,19 @@ def test_insight_retention_hogql(self) -> None:
     def test_insight_with_filters_via_hogql(self) -> None:
         filter_dict = {"insight": "LIFECYCLE", "events": [{"id": "$pageview"}]}
 
-        Insight.objects.create(
+        insight = Insight.objects.create(
             filters=Filter(data=filter_dict).to_dict(),
             team=self.team,
             short_id="xyz123",
         )
 
         # fresh response
-        response = self.client.get(f"/api/projects/{self.team.id}/insights/?short_id=xyz123")
+        response = self.client.get(f"/api/projects/{self.team.id}/insights/{insight.id}/?refresh=true")
```

Comment:
Hmm.. this is actually a different endpoint (I call it "numeric insights api" 😆 ) and we use the `?short_id=xyz123` frontend side when opening a saved insight.

Reply:
Yup, but I _think_ this test was a bit off, because others use the numeric endpoint – only this one supports `refresh`